test/api/webSockets.test.js

```diff
@@ -85,6 +87,7 @@ describe('webSockets', () => {
 
 		assert.ok(err);
 		assert.strictEqual(webSockets.isConnected(), false, 'connected');
+		console.error.restore();
```

Comment:
If test fails during execution this line may not be called, probably better do this in after or afterEach